Produlytics/ui/front-end/src/app/main/selection/selection.component.ts

```diff
@@ -1,29 +1,66 @@
-import {Component, OnInit, ViewEncapsulation} from '@angular/core';
-import { CharacteristicNode } from '../device-selection/selection-data-source/characteristic-node';
-/**
- * Component la cui view racchiude la selezione dei grafici e la loro rappresentazione
- */
+import { Component, OnInit, Output } from '@angular/core';
+import { FlatTreeControl } from "@angular/cdk/tree";
+import {
+  UnarchivedCharacteristicAbstractService
+} from "../../model/characteristic/unarchived-characteristic-abstract.service";
+import { UnarchivedDeviceAbstractService } from "../../model/device/unarchived-device-abstract.service";
+import { SelectionNode } from './selection-data-source/selection-node';
+import { SelectionDataSource } from './selection-data-source/selection.data-source';
+import { EventEmitter } from '@angular/core';
+import { CharacteristicNode } from './selection-data-source/characteristic-node';
+
+// TODO STA CLASSE È TUTTA DA RINOMINARE AAAAAAA (device->characteristic)
```

Comment:
È da fare in questa PR?

Reply:
> È da fare in questa PR?

Questa classe la ha scritta @MicheGit anche se mi pareva fosse a posto

Reply:
No non ora dovremmo cambiare anche tutta la progettazione

Reply:
Allora scegli un TODO un po' più presentabile perchè questa repo sarebbe pubblica e magari Cardin ci da un'occhiata.